grakn-core/src/main/java/ai/grakn/concept/LabelId.java

```diff
@@ -34,49 +35,24 @@
  *
  * @author fppt
  */
-public class LabelId implements Comparable<LabelId>, Serializable {
-    private static final long serialVersionUID = 3181633335040468179L;
-
-    private Integer typeId;
-
-    private LabelId(Integer typeId){
-        Preconditions.checkNotNull(typeId);
```

Comment:
It disallows them, unless you mark the method as `@Nullable`, so we're exactly as safe as before 👍